### PR TITLE
Registry connection complete

### PR DESCRIPTION
**Pull request by abenezer-gelaw**
This PR completes the clustering part of the tool! main.py takes the project name we want to compute clusters for, extract and cleans the words, forms clusters using dbscan to make WordGroups, and finally uploads the WordGroups to the registry server with a call that forms a unique ID for each word group. This ID satisfies the regex requirements of the registry server. It has been tested on 14,000 words extracted from OpenAPI's googleapis.com and subsets of them.

### CODE REVIEW

File: consistency/tool/clustering/clustering.py [hunk 1]
```diff
@@ -8,6 +8,8 @@
 from google.cloud.apigeeregistry.applications.v1alpha1.consistency import (
     word_group_pb2 as wg,
 )
+import timeit
```
Comment (shrutiparabgoogle) [inline consistency/tool/clustering/clustering.py:11]: Is this still used anywhere?

File: consistency/tool/comparison/comparison.py [hunk 2]
```diff
@@ -0,0 +1,32 @@
+class Comparison:
```
Comment (shrutiparabgoogle) [inline consistency/tool/comparison/comparison.py:1]: I am fine with merging this and iterating over this structure as we develop the comparison part of the tool.

**Review by shrutiparabgoogle — APPROVED** [review]
LGTM with one minor comment.

File: consistency/tool/clustering/clustering.py (hunk 1 above)
Comment (abenezer-gelaw) [reply consistency/tool/clustering/clustering.py:11]: Whoops, overlooked. Removed!

File: consistency/tool/clustering/main.py [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+            + "/locations/global/artifacts/wordgroup-"
+            + id,
+            mime_type="application/octet-stream;type=google.cloud.apigeeregistry.applications.v1alpha1.consistency.WordGroup",
+            contents=word_group.SerializeToString(),
+        )
+
+        createArtifactRequest = rs.CreateArtifactRequest(
+            parent="projects/" + project_name + "/locations/global",
+            artifact=artifact,
+            artifact_id="wordgroup-" + id,
```
Comment (Krpate) [inline consistency/tool/clustering/main.py:61]: It maybe unrelated to this CL but as we are generating the artifact ID from the cluster ID how would we handle if the cluster ID representing the cluster changes while recomputing clusters? Would we delete the old word group artifact?

**Review by Krpate — APPROVED** [review]

File: consistency/tool/clustering/main.py (hunk 3 above)
Comment (abenezer-gelaw) [reply consistency/tool/clustering/main.py:61]: My understanding is that at each re-computation of clusters, we delete all wordgroups we have stored first. @shrutiparabgoogle may clarify here.

Comment (shrutiparabgoogle) [reply consistency/tool/clustering/main.py:61]: Yes, that is correct cluster re-computation will be delete and create initially, next phase can be to make it smart by deleting only non-existent clusters and updating the existing ones.

**Pull request merged**